examples/coexist_klog_v1/coexist_klog_v1.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogWriter struct{}
+
+func (kw klogWriter) Write(p []byte) (n int, err error) {
+	if p[0] == 'I' {
+		klog.InfoDepth(klogv2.OutputCallDepth, string(p[klogv2.DefaultPrefixLength:]))
+	} else if p[0] == 'W' {
+		klog.WarningDepth(klogv2.OutputCallDepth, string(p[klogv2.DefaultPrefixLength:]))
+	} else if p[0] == 'E' {
+		klog.ErrorDepth(klogv2.OutputCallDepth, string(p[klogv2.DefaultPrefixLength:]))
+	} else if p[0] == 'F' {
+		klog.FatalDepth(klogv2.OutputCallDepth, string(p[klogv2.DefaultPrefixLength:]))
+	}
+	return len(p), nil
+}
+
+func main() {
+	klog.InitFlags(nil)
+	klogv2.LogToStderr(false)
+	klogv2.SetOutputBySeverity("INFO", klogWriter{})
```

Comment:
only these two lines are what they would need to add to their existing klog setup that is working the way they want in order to wire klog/v2 -> klog/v1, right?

should probably make it clear the `klog.InitFlags(nil)` and all the flag.Set stuff in this example only affects klog v1 and can be ignored if they already have klog v1 working as they want

Reply:
also note that skipHeaders means the output writer doesn't get severity prefix